fixtures/src/components/uncontrolled/grouped-character.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import GroupedCharacter from 'carbon-react/lib/__experimental__/components/grouped-character';
+import { LogConsumer } from '../log';
+
+const UncontrolledGroupedCharacter = () => {
+  return (
+    <LogConsumer>
+      {(log) => {
+        const onChange = (e) => {
+          log(e, { method: 'onChange' });
+        };
+        const onBlur = e => log(e, { method: 'onBlur' });
+        return (
+          <React.Fragment>
+            <div id='controlled_grouped_character'>
+              <h1>Uncontrolled GroupedCharacter</h1>
+              <ul>
+                <li>onChange handler should update the log when the value is changed, e.target.value should be the
+                  users input
```

Comment:
I think there should be an explanation about `e.target.value` containing `rawValue` and `formattedValue`